File: PyLibs/src/Libs/Log/LogTypeId.py
# -*- coding: utf-8 -*-

from Libs.Core.GObject import GObject;

class LogTypeId(GObject):
    eLogCommon = 0;         # 通用日志
    eLogSceneCull = 1;      # 场景裁剪日志
    eLogMSceneManager = 2;  # 场景管理器
    eLogTest = 3;           # 测试日志

File: JSLib/src/Log/LogTypeId.js
namespace SDK.Lib
{
    public enum LogTypeId
    {
        eLogCommon,         // 通用日志
        eLogSceneCull,      // 场景裁剪日志
        eLogMSceneManager,  // 场景管理器
        eLogTest,           // 测试日志
        eLogTestRL,         // 测试资源加载
        eLogResLoader,      // 资源加载
        eLogLocalFile,      // 本地文件系统， MDataStream
    }
}